tests/cases/distributed/explain/subqueries.result

```diff
@@ -113,12 +113,11 @@ EXPLAIN INSERT INTO other SELECT i, 2 FROM integers WHERE i=(SELECT MAX(i) FROM
 | logical_plan | Dml: op=[Insert] table=[other]                                    |
 |              |   Projection: integers.i AS i, TimestampMillisecond(2, None) AS j |
 |              |     Inner Join: integers.i = __scalar_sq_1.MAX(integers.i)        |
-|              |       Projection: integers.i                                      |
-|              |         MergeScan [is_placeholder=false]                          |
+|              |       TableScan: integers projection=[i]                          |
 |              |       SubqueryAlias: __scalar_sq_1                                |
-|              |         Aggregate: groupBy=[[]], aggr=[[MAX(integers.i)]]         |
-|              |           Projection: integers.i                                  |
-|              |             MergeScan [is_placeholder=false]                      |
```

Comment:
As I said before, the subquery here brings `MergeScan`, which pushes down `Projection` and `Aggregate`. Changing to `TableScan` can't even be executed.

Reply:
How about making the push-down plan visible?

Reply:
Nah there won't be an actual `TableScan`. We execute DML in two steps, but this isn't reflected in the explained plan.

>How about making the push-down plan visible?

This is on my list. I'm finding a way to display it prettier.

Reply:
If so, explain of dml is different from the actual plan :\

Reply:
> subquery in dml will be executed separately as a query in `execute_dml_subquery`

But it has the wrong logical plan. It still cannot be executed for a plan with `DistTable`. `execute_dml_subquery` doesn't do the plan rewriting.

Reply:
I made it too complicated. The only rule we need is to avoid the non-`TableScan` plan and apply all other cases

Reply:
Sorry for my mistake, I always thought that the optimization of logical plan is done before execution, but in fact it is done during execution...